cmd/kaf/consume.go

```diff
@@ -309,10 +311,20 @@ func handleMessage(msg *sarama.ConsumerMessage, mu *sync.Mutex) {
 	if !raw {
 		if isJSON(dataToDisplay) {
 			dataToDisplay = formatValue(dataToDisplay)
+		} else if trimMessageHeaderBytes > 0 {
```

Comment:
Should we always respect the set trim? Probably fine to attempt without first.

Reply:
In the protobuf case, that would result in trimming twice

Reply:
I modified when the trimming occurs to always respect the value passed in, and with the side effect of making the code cleaner!